sdk/communication/communication-chat/CHANGELOG.md

```diff
@@ -6,6 +6,13 @@
 
 - Prohibit sending the typing notification request within 8 seconds of the previous request.
 
+### Features Added
+
+- Updated to @azure/communication-signaling@1.0.0-beta.12.
+- Added two new APIs:
+  - on(event: "realTimeNotificationConnected", listener: () => void): void;
+  - on(event: "realTimeNotificationDisconnected", listener: () => void): void;
```

Comment:
Can we consolidate this into a bullet that talks about the events and what they do? e.g. "Added two new events `realTimeNotificationConnected` and `realTimeNotificationDisconnected` that allow the developer to know when the connection to the call server is active"

Reply:
consolidated